CHANGELOG.md

```diff
@@ -2,14 +2,9 @@
 All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org)
 
 ## MASTER
-### Added
-- Added `region_label` field to the output of `site:info`, labeled "Region". (#1982)
-- Added `region_label` field to the output of `site:list`, labeled "Region". (#1982)
-- Added `region_label` field to the output of `Site::serialize()`. (#1982)
-
 ### Changed
-- `site:info`'s label of the `region` field has been changed from "Region" to "Region Code". (#1982)
-- `site:list`'s label of the `region` field has been changed from "Region" to "Region Code". (#1982)
+- `site:info`'s value of the `region` field has been changed to use human-readable region names. (#1985)
```

Comment:
I do not like our replacing the data we had with new data and using the same name as before. This would constitute a breaking change.